### DISCUSSION TITLE
Issue with Development Branch  #1997

### DISCUSSION THREAD

**Discussion by HathewayWill** [dtcenter/METplus#2019] [Installation] [answered]
Tried installing with development branch compile_met_all.sh

Unsuccessful install

Attached are relevant logs for @georgemccabe 

[compile_MET_all.log](https://github.com/dtcenter/METplus/files/10485529/compile_MET_all.log)
[config.log](https://github.com/dtcenter/METplus/files/10485530/config.log)
[configure.log](https://github.com/dtcenter/METplus/files/10485531/configure.log)
[make.log](https://github.com/dtcenter/METplus/files/10485532/make.log)

**Comment by jprestop** [comment]
HI @HathewayWill.

In your make.log file, I see:

> g++: error: unrecognized command-line option ‘-r -ldl -lutil -lm’
g++: error: unrecognized command-line option ‘-r -ldl -lutil -lm’

Checking the value of your PYTHON variables, in your config.log file, I see:

> MET_PYTHON_CC='-I/usr/include/python3.10'
MET_PYTHON_LD='-L/usr/lib/python3.10/config-3.10-x86_64-linux-gnu\ -L/usr/lib\ -lpython3.10\ -lcrypt\ -lpthread -r\ -ldl\ -lutil\ -lm'

I'm not familiar with the "-r" in the setting of MET_PYTHON_LD.  Could you please send the output from "python3-config --ldflags"?  You could also try removing the "-r" from MET_PYTHON_LD and try rerunning.

**Comment by HathewayWill** [comment, marked as answer]
@jprestop @georgemccabe @JohnHalleyGotway 

So it was the -lpthread -r that was the issue.

For reference with Make_ARGS=-j 32:
 MET compiles and installs in 2 minutes 35 seconds.

Without the minus -j option
MET compiles and installs in  8 minutes 5 seconds.

So that's a 313% improvement on compilation time.
